src/ai/backend/manager/api/session.py

```diff
@@ -186,6 +194,14 @@ def check_and_return(self, value: Any) -> object:
 creation_config_v5 = t.Dict({
     t.Key("mounts", default=None): t.Null | t.List(t.String),
     tx.AliasedKey(["mount_map", "mountMap"], default=None): t.Null | t.Mapping(t.String, t.String),
+    tx.AliasedKey(["mount_options", "mountOptions"], default=None): t.Null
+    | t.Mapping(
+        t.String,
+        t.Dict({
+            t.Key("type", default=MountTypes.BIND): MountTypes,
+            tx.AliasedKey(["permission", "perm"], default=None): t.Null | MountPermission,
```

Comment:
Though this issue is from a missing refactoring, `MountPermission` and `MountTypes` are not a trafaret. I think we need to make trafaret wrappers of these enums.  We could update the base type from `(str, enum.Enum)` to `enum.StrEnum` (introduced in 3.11) and wrap these with `t.Enum`.

> [!NOTE]
> A trafaret should call `self._failure()` to generate a properly structured `trafaret.DataError` to be reported as an input valiadtion error. See custom trafaret impl. examples in the `ai.backend.common.validators` module.